tests/plugins/cloudservices/unit.js

```diff
@@ -81,6 +81,31 @@ bender.test( {
 			listener.removeListener();
 		}
 	},
+	'test no URL error': function() {
+		var instance = new this.cloudservices.cloudServicesLoader( this.editor, mockBase64, null, 'different_token' ),
+		listener = this.editor.once( 'fileUploadRequest', this.commonRequestListener, null, null, 0 );
+		this.editor.config.cloudServices_url = undefined;
+		CKEDITOR.once( 'log', function( evt ) {
+			evt.cancel();
+			assert.areEqual( 'cloudservices-url-error', evt.data.errorCode, 'There should be URL error log.' );
+		} );
+		instance.upload();
+		this.editor.config.cloudServices_url = 'cs_url';
```

Comment:
Why do you need to change this config on the end of the test? Maybe setting `cs_url` and `cs_token` could be done in `setUp` so it will be always set when test is executed (if it is really needed)?